### PR TITLE
Hello World App - MongoDB Atlas and App Engine Flex - NodeJS

### PR DESCRIPTION
**Pull request by ideafest**
Hi,

I am customer engineer at google (LDAP: arajwade@) working with MongoDB and I have created a tutorial on how to connect with MongoDB's Cloud Database "Atlas".

I would like to add this tutorial to GCP community tutorials.

Please ping me you have any questions related to this.

Regards,
Abhijeet

- [x] Tech writing / styleguide review - @ryanmcginnis    
- [x] Formatting review for publishing - @jmdobry

### CODE REVIEW

**Comment by ideafest** [comment]
I did one more commit from updated email.

**Comment by ryanmcginnis** [comment]
I don't have cycles today, but I can do this tomorrow.

File: tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
+## "Hello World” App - MongoDB Atlas and App Engine Flex - NodeJS
+author: arajwade, ksmith
+tags: App Engine, Node.js, MongoDB Atlas
+date_published: 2018-05-06
+
+## Overview: 
+In this tutorial, we will be building a “Hello World” application using Node JS with Google App Engine Flex for our frontend and a MongoDB Atlas multi-regional cluster on GCP as our primary database.
```
Comment (ryanmcginnis) [inline tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md]: Change "we will be" to "you will be"

File: tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+
+### Technical Complecity: 
+Beginner
+
+### Duration: 
+45 Minutes
+
+### Objective:
+1. Create and configure MongoDB Atlas multi-regional cluster on GCP
```
Comment (ryanmcginnis) [inline tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md]: For this "Objective" list:

- Change to "Objectives"
- Change "our" to "your"
- Add a period to each list item
- Change semicolon in item 5 to a comma

File: tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+![image](https://github.com/ideafest/tutorials/blob/master/image3.png)
+
+3. Enter necessary details and click “Continue”.
+![image](https://github.com/ideafest/tutorials/blob/master/image12.png)
+
+4. Once on the MongoDB Atlas Homepage, select “Build a New Cluster”
+![image](https://github.com/ideafest/tutorials/blob/master/image15.png)
+
+5. Create New Cluster by selecting Google Cloud Platform as a “Cloud Provider”. Next select the region where you want to place your Atlas cluster.  Ideally, your cluster will be located close to your end user for lower latency. Note that we can select a free tier region in your area of choice, as noted by the “Free Tier Available” icon  This is a no-cost option to get started. However, since we are assuming our end users will span across the globe, we want our Atlas cluster to be multi-regional.   To enable a multi-regional cluster, we will need to select a M10 or larger cluster size. So for now, for our use case, let’s assume our primary end users for our application will be based in the US and a smaller percentage based in the UK and Australia.  As such, we will select our primary region to be in North America, based in Iowa (us-central1).
```
Comment (ryanmcginnis) [inline tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md]: In list item 5, add a period after "'Free Tier Available' icon'"

File: tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+
+7. Now that we selected a M10 cluster, let's go back to “Cloud Provider & Region” and toggle the option to “Configure clusters across multiple regions” from “No” to “Yes”. Once we enable this we will see more options.  You will see your previously selected region as "Preferred" under “Node Type”. Since for our use-case we will also have some users in the UK and Australia, we want to take into the considering the distance between our primary cluster in the US and enable lower latency for better read performance. As such, we will add a Read-only replica in London (europe-west2) and Sydney (australia-southeast1). Under the subheading “Deploy read-only replicas”, select “Add a node” and add these two regions with 1 node each.
+![image](https://github.com/ideafest/tutorials/blob/master/image27.png)
+
+8. We can skip the section “Additional settings” by clicking the “NEXT: CLUSTER NAME” button since we will keep the default settings.
+
+![image](https://github.com/ideafest/tutorials/blob/master/image35.png)
+
+9. Under “Cluster Name”- Provide a Cluster Name. For this demo, we will keep the default of “Cluster0”. Click on “Create Cluster” button at the bottom of the page.
```
Comment (ryanmcginnis) [inline tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md]: In list item 9, change to "'Cluster Name', provide a name for your cluster."

File: tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+
+![image](https://github.com/ideafest/tutorials/blob/master/image38.png)
+
+10. Our cluster is spinning up...
+
+![image](https://github.com/ideafest/tutorials/blob/master/image36.png)
+
+11. While this spins up, lets click on “Security” tab and then click on “Add New User”.
```
Comment (ryanmcginnis) [inline tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md]: Item 11: change to "let's"

File: tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+![image](https://github.com/ideafest/tutorials/blob/master/image19.png)
+
+18. Click on “I am using driver 3.4 or earlier” and copy the connection string and keep it in a text file. We will be using it in our node JS application to connect to MongoDB Atlas in Part 2 of this document. Close the pop-up dialogue
+  
+![image](https://github.com/ideafest/tutorials/blob/master/image40.png)
+![image](https://github.com/ideafest/tutorials/blob/master/image41.png)
+
+### Part 2: Configuring our Google Cloud Platform Account
+
+1. Go to cloud.google.com and login with your google account. If you don't have a google account, please create a free trial account by following instructions at this [link](https://console.cloud.google.com/freetrial).
```
Comment (ryanmcginnis) [inline tutorials/mongodb-atlas-appengineflex-nodejs-app/index.md]: Capitalize "Google" throughout the page.

Delete "please" throughout the page.

**Comment by ideafest** [comment]
Merged the changes.

**Pull request merged**